replication-commands/src/main/java/org/codice/ditto/replication/commands/ConfigDeleteCommand.java

```diff
@@ -91,6 +115,41 @@
   private static final int DEFAULT_START_INDEX = 0;
 
   @Override
+  protected final Object executeWithSubject() throws Exception {
+    if (configNames == null || configNames.isEmpty()) {
+      if (deleteMetacards) {
+        removeAllReplicatedMetacards();
+        removeReplicationItems(null);
+      }
+
+      if (deleteHistory) {
+        removeHistoryItems(null);
+      }
+
+      if (deleteConfig) {
+        replicatorConfigLoader
+            .getAllConfigs()
+            .stream()
+            .forEach(config -> replicatorConfigLoader.removeConfig(config));
+      }
+    } else {
+      for (final String configName : new HashSet<>(configNames)) {
+        final Optional<ReplicatorConfig> config = replicatorConfigLoader.getConfig(configName);
+
+        if (config.isPresent()) {
```

Comment:
✏️ As opposed to first testing `isPresent() followed by `.get()`, one approach is to do `.orElse()` to get `null` or a config and then simply testing `!= null`

Reply:
Done